### PR TITLE
Fix Bug Initializing Tags

### PR DESCRIPTION
**Pull request by zenine07**
#  #

## 🗣 Description ##

Some tags were no longer used and needed to be eliminated. Found that the tag initialization process had it's own set of code to create faker tags. Removed that function and pointed to the shared module.

Also added @dylanj1752 and @itsmostafa as codeowners.

## 🧪 Testing ##

Tested locally and made sure old tags were replaced. All automated tests were also passing.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [ ] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
* [ ] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [ ] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
* [ ] Tests have been added and/or modified to cover the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
👍 LGTM, though I have some feedback for you when creating future PRs:
* The "Description" section should contain info about "what" was done.  Anything related to "why" it was done belongs in the "Motivation and context" section.  Neither of those two sections should ever be left blank.
* If you have no screenshots (which is common), you should delete that section header from the PR.
* Similarly, some of the checkboxes in the "Checklist" section may not apply to your PR and can be deleted.  This applies to 3 of the 4 that are currently unchecked (note that you should check the box for `All relevant type-of-change labels have been added.` since you added the `bugfix` label already).  All of the checkboxes that you checked are required and should never be deleted from the PR.  The goal of those checkboxes is to have every box checked before a PR is considered "ready for approval".

**Comment by dav3r** [comment]
Also, you should use our standard "bug" label for this PR, not "bugfix".  Our intent is to have "bug" cover both issues where bugs are reported as well as PRs that fix those bugs.  

Our standard set of labels can be found here:  https://github.com/cisagov/github-label-management/labels

**Review by itsmostafa — APPROVED** [review]

**Review by jsf9k — COMMENTED** [review]
Looks good to me, except for what @dav3r pointed out.  I removed your "bugfix" label and replaced with our standard "bug" label.

I'm only commenting because I'm not really qualified to pass judgement on the main part of this PR.  The codeowners change looks fine to me, though.

**Pull request merged**
